src/toil/utils/toilStatus.py

```diff
@@ -49,14 +49,14 @@ def print_dot_chart(self) -> None:
 
         # Make job IDs to node names map
         jobsToNodeNames: Dict[str, str] = dict(
-            map(lambda job: (str(job.jobStoreID), job.jobName), self.jobsToReport)
+            map(lambda job: (str(job.jobStoreID), str(job.jobStoreID).replace("_", "___").replace("/", "_").replace("-", "__")), self.jobsToReport)
```

Comment:
Hmmm... `name_/something` seems equivalent to `name/_something` after conversion.  Is that alright?

Reply:
Oh good point, I didn't think about adjacent sequences.